doc/misc/undocumented-preferences.md

```diff
@@ -0,0 +1,20 @@
+# Why the following scanner preferences are undocumented
+
+Some scanner preferences are found in the scanner code but are not documented in the man page. This is because such preferences are for internal usage or just preferences to be set via the client, and there is not intention to be set via the configuration file. The following are the preferences and what are they used for, and the reason why are not documented in the man page.
```

Comment:
```suggestion
Some scanner preferences are found in the scanner code but are not documented in the man page. This is because such preferences are for internal usage or just preferences to be set via the client and there is no intention to be set via the configuration file. The following are the preferences and what they are used for and the reason they are not documented in the man page.
```